.github/workflows/python-format-CI.yml

```diff
@@ -0,0 +1,20 @@
+name: CI Format Check
+
+on:
+  pull_request:
+    paths:
+      - 'tools/chapel-py/**/*.py'
+      - 'tools/chplcheck/**/*.py'
+      - 'tools/chpl-language-server/**/*.py'
+  workflow_dispatch:
+
+jobs:
+  check-format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: psf/black@stable
+        with:
+          options: "--check --verbose"
```

Comment:
Noting that black defaults to a line length of 88, is that the intended line width?